game/ffmpeg.cc

```diff
@@ -301,6 +364,8 @@ AudioFFmpeg::AudioFFmpeg(fs::path const& filename, int rate, AudioCb audioCb) :
 
 double FFmpeg::duration() const { return double(m_formatContext->duration) / double(AV_TIME_BASE); }
 
+double FFmpeg::replayGain() const { return m_replaygain; }
```

Comment:
Name it `getReplayGain`.

Reply:
Yes, I am. `duration()` is stl-style and was widely used in performous. But it is expiring and the `get-` and `set-` notation is used.